package.json

```diff
@@ -72,19 +72,19 @@
     "@comandeer/eslint-config": "^0.1.0",
     "@comandeer/is-ci": "^1.0.0",
     "@comandeer/rollup-lib-bundler": "^0.6.0",
-    "babel-preset-env": "^1.6.1",
```

Comment:
If the PR is intended to change only Babel version, then please narrow changes only to `@babel/core` package, leaving all other untouched.